pkg/collection/entry/entry.go

```diff
@@ -12,8 +12,9 @@ import (
 )
 
 var (
-	_                  = collection.Entry(&Entry{})
-	serializedDataSize = swarm.SectionSize * 2
+	_                           = collection.Entry(&Entry{})
+	serializedDataSize          = swarm.SectionSize * 2
```

Comment:
what is this meant to be?

Reply:
Entry stores two references, one for metadata and other for the actual data. so it is multiplied by 2.  But for Encrypted file, the references will be 64 ... that was added as a new constant.